db/migrate/20200405233913_add_no_sp_tracking_to_doc_auth_logs.rb

```diff
@@ -0,0 +1,8 @@
+class AddNoSpTrackingToDocAuthLogs < ActiveRecord::Migration[5.1]
+  def change
+    safety_assured do
+      add_column :doc_auth_logs, :no_sp_at, :datetime
```

Comment:
Could we change this name to be a little more descriptive? Maybe something like `no_sp_session_started_at`?